core/src/server/component.cpp

```diff
@@ -128,6 +128,9 @@ additionalProperties: false
                     cert:
                         type: string
                         description: path to TLS certificate
+                    cert-chain:
+                        type: string
+                        description: path to TLS certificates chain
```

Comment:
Please add `defaultDescription: ''` to highlight that the valie is optional

Reply:
added